modules/gameResult.js

```diff
@@ -0,0 +1,38 @@
+import { Console } from '@woowacourse/mission-utils';
+import { MESSAGE } from './constant.js';
+
+const findWhoIsWinner = (calcuratedScoreBoard) => {
+  const winner = [];
+
+  calcuratedScoreBoard.forEach((car, index) => {
+    const winnerLastIndex = winner[winner.length - 1]?.score.length;
+    if (index === 0) {
+      winner.push(car);
+      return;
+    }
+
+    if (car.score.length > winnerLastIndex) {
+      winner.length = 0;
+      winner.push(car);
+      return;
+    }
+
+    if (car.score.length === winnerLastIndex) {
+      winner.push(car);
+    }
+  });
+
+  return winner;
+};
```

Comment:
배열을 만들고 [ ] push 하는 방식은 명령형 프로그래밍에 가깝다고 생각해서 reduce 를 활용해서 
한번에 우승자를 구하는 방법도 고려해보시면 좋을 것 같습니다! 
winnerLastIndex 에 우승자 최고점을 담고 있기 때문에 인덱스라는 표현은 없애고 점수같은 다른 표현으로 대체하면 좋을 것 같다고 생각했습니다! ㅎㅎ 

```suggestion
const findWhoIsWinner = (calcuratedScoreBoard) => {
  const winner = [calcuratedScoreBoard[0]];

  calcuratedScoreBoard.forEach((car, index) => {
    const winnerScore = winner[0].score.length;

    if (car.score.length > winnerScore ) {
      winner= [car]
      return;
    }

    if (car.score.length === winnerScore ) {
      winner.push(car);
    }
  });

  return winner;
};
```

Reply:
앗... !!

이렇게 또 리팩토링 해주셔서 감사드립니다... 안그래도 이 코드를 작성하고 마음에 안들어 리팩토링을 위해 뚫어지게 쳐다 봤지만 별다른 방도를 찾지 못했는데 이렇게 간단히 해결하시니.... 아직 제가 많이 부족한 탓입니다...

reduce를 이용한 방법도 다시한번 고안해 보겠습니다 !
친절한 리뷰 감사드립니다 !!

ps. 서연님께서 제안해주신 코드에서 winner = [car] 새로운 배열이 할당되는거라 winner가 let으로 선언되야하지 않을까 합니다 !